<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poisson Image Blending</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="ImageGraphicsView" name="graphicsViewSource">
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="transferButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ImageGraphicsView" name="graphicsViewTarget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_source_image"/>
    <addaction name="actionOpen_target_image"/>
    <addaction name="actionSave_project"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionExport_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionTemp_Test"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_source_image">
   <property name="text">
    <string>Open source image...</string>
   </property>
  </action>
  <action name="actionOpen_target_image">
   <property name="text">
    <string>Open target image...</string>
   </property>
  </action>
  <action name="actionExport_as">
   <property name="text">
    <string>Export as...</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTemp_Test">
   <property name="text">
    <string>Temp Test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>imagegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
